src/query/interpreter.cpp

```diff
@@ -2605,6 +2616,10 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
   const auto trimmed_query = utils::Trim(upper_case_query);
 
   if (trimmed_query == "BEGIN" || trimmed_query == "COMMIT" || trimmed_query == "ROLLBACK") {
+    if (trimmed_query == "BEGIN") {
+      Statistics::IncrementCounter(Statistics::ActiveTransactions);
+    }
```

Comment:
PrepareTransactionQuery method already does this check whether the trimmed_query is BEGIN